packages/main/src/TimePickerBase.ts

```diff
@@ -75,16 +75,38 @@ import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverComm
 *
 * @event sap.ui.webc.main.TimePickerBase#change
  * @public
- */
-@event("change")
+ * @param {string} value The submitted value.
+ * @param {boolean} valid Indicator if the value is in correct format pattern and in valid range.
+*/
+@event("change", {
+	detail: {
+		value: {
+			type: String,
+		},
+		valid: {
+			type: Boolean,
+		},
+	},
+})
 
 /**
  * Fired when the value of the <code>ui5-time-picker</code> is changed at each key stroke.
  *
  * @event sap.ui.webc.main.TimePickerBase#input
  * @public
- */
-@event("input")
+ * @param {string} value The submitted value.
```

Comment:
If above in `change`, `value` is "The submitted value", perhaps here in `input` it's something else, the live value, the typed value. Or, they are both described as "The submitted value" on purpose?

Reply:
It was directly taken from the DatePicker as the cases are similar. Will change it with "current".